e2e/task/task_test.go

```diff
@@ -0,0 +1,131 @@
+package task
+
+import (
+	"github.com/aws/copilot-cli/e2e/internal/client"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Task", func() {
+	Context("when creating a new app", func() {
+		var err error
+		BeforeAll(func() {
+			_, err = cli.AppInit(&client.AppInitRequest{
+				AppName: appName,
+			})
+		})
+
+		It("app init succeeds", func() {
+			Expect(err).NotTo(HaveOccurred())
+		})
+	})
+
+	Context("when creating a new environment", func() {
+		var (
+			err error
+		)
+		BeforeAll(func() {
+			_, err = cli.EnvInit(&client.EnvInitRequest{
+				AppName: appName,
+				EnvName: envName,
+				Profile: "default",
+				Prod:    false,
+			})
+		})
+
+		It("env init should succeed", func() {
+			Expect(err).NotTo(HaveOccurred())
+		})
+	})
+
+	Context("when running in an environment", func() {
+		var err error
+		BeforeAll(func() {
+			_, err = cli.TaskRun(&client.TaskRunInput{
+				GroupName: groupName,
+
+				Dockerfile: "./backend/Dockerfile",
+
+				AppName: appName,
+				Env: envName,
+			})
+		})
+
+		It("should succeed", func() {
+			Expect(err).NotTo(HaveOccurred())
+		})
+	})
+
+	Context("when running in default cluster and subnets", func() {
```

Comment:
We should validate that there is indeed a "default" cluster in the account where the task stack is created before merging the PR

Reply:
OK created one :) so we should be good here.

Reply:
hahaha okay I just opened my IDE